llama_index/data_structs/data_structs.py

```diff
@@ -163,9 +163,9 @@ def get_type(cls) -> IndexStructType:
 class IndexDict(IndexStruct):
     """A simple dictionary of documents."""
 
-    # mapping from vector store id to node id
+    # mapping from vector store id to node doc_id
     nodes_dict: Dict[str, str] = field(default_factory=dict)
-    # mapping from doc_id to vector store id
+    # mapping from node doc_id to vector store id
     doc_id_dict: Dict[str, List[str]] = field(default_factory=dict)
```

Comment:
nit: I don't think we need this field anymore? I think it was previously used to keep the mapping from ref_doc_id to (node) doc_ids (via vector_store_id, soooo confusing). But since you are maintaining the mapping outside in docstore, I think we no longer need to keep track of that mapping here.